spec/models/order_spec.rb

```diff
@@ -17,35 +16,87 @@
   end
 
   describe '#fulfilled?' do
-    # These are gross, I want to use one-liner syntax
     subject { build(order_type) }
 
+    it { is_expected.to_not be_fulfilled }
+
     context 'when fulfilled_at is set' do
-      it 'it is fulfilled' do
+      it 'is fulfilled' do
         subject.fulfilled_at = Time.now
 
         expect(subject).to be_fulfilled
       end
     end
+  end
+
+  describe '#canceled?' do
+    subject { build(order_type) }
+
+    it { is_expected.to_not be_canceled }
+
+    context 'when status is set to canceled' do
+      it 'is canceled' do
+        subject.status = 'canceled'
```

Comment:
Interesting point, I was kinda wondering about that. Having read http://betterspecs.org/ a few times I'd always kinda just assumed where it says about using let and let! that it also ment 'don't use before' but really it's saying don't use before { @blah = 'blah' }.